src/breadcrumbs/_base.scss

```diff
@@ -2,40 +2,29 @@
  * @file Breadcrumbs
  * @module breadcrumbs
  * @overview Styles for Casper breadcrumbs
- * @TODO: expand/solidify styling, here they have styles to support grey and blue (like on the main site)
+ * @TODO: create set for footer breadcrumbs
```

Comment:
Agreed here. One way to think about this pattern is a breadcrumbs on ligtht background and breadcrumbs on dark background. Maybe something like `breadcrumbs--light-background` and `breadcrumbs--dark-background` but with better modifiers